packages/analytics-js-plugins/src/deviceModeDestinations/utils.ts

```diff
@@ -187,9 +187,10 @@ const getCumulativeIntegrationsConfig = (
         normalizeIntegrationOptions(dest.instance?.getDataForIntegrationsObject()),
       );
     } catch (err) {
-      logger?.error(
-        DESTINATION_INTEGRATIONS_DATA_ERROR(DEVICE_MODE_DESTINATIONS_PLUGIN, dest.userFriendlyId),
+      errorHandler?.onError(
```

Comment:
I though we had agree for device mdoes to just log the errors and not throw then for reporting